.github/workflows/build-webapi.yaml

```diff
@@ -9,27 +9,45 @@ on:
     branches: [ dev, main ]
     paths:
       - 'src/dotnet/**'
+      - '.github/workflows/**'
 
 jobs:
-  build:
+  container-build-step:
     runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: ./src/dotnet/CarbonAware.WebApi
     steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+    
+    - name: Docker Target Build
+      run: docker build --target=build . -f CarbonAware.WebApi/Dockerfile -t ca-api-build
```

Comment:
We've talked about changing the folder structure at some point to make it easier to understand and navigate. Is this path unnecessarily making this "brittle" for that future? Is there a way we can highlight or parameterize these types of variables?

Reply:
Done!